resources/views/cart/overview.blade.php

```diff
@@ -7,7 +7,7 @@
 @section('content')
     <cart v-cloak>
         <div class="container" v-if="hasItems" slot-scope="{ cart, hasItems, changeQty, remove }">
-            <div class="flex max-md:flex-col text-sm text-primary">
+            <div class="flex max-md:flex-col text-sm text-primary flex-wrap gap-x-8">
                 <div class="flex-1">
                     @include('rapidez-ct::cart.cart')
```

Comment:
Can you create a component of this whole section? with 2 slots 
- 1 for the flex-1 div
- and 1 for the sidebar div,

this is because this exact same layout is being used in the checkout, and this is the cart. And in the future we will be applying this exact layout in the customer centre, so it would help me allot if i can use your component for the layout